weaviate/tests/common.py

```diff
@@ -0,0 +1,111 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+
+from datadog_checks.base import is_affirmative
+
+USE_AUTH = is_affirmative(os.environ.get('USE_AUTH'))
+
+MOCKED_INSTANCE = {
+    'openmetrics_endpoint': 'http://weaviate:2112/metrics',
+    'weaviate_api_endpoint': 'http://weaviate:8080',
+    'tags': ['test:tag'],
```

Comment:
Do we have a test where we only provide the openmetrics_endpoint and another with only the weaviate_api_endpoint ?